Jellyfin.Plugin.AniList/Providers/AniList/AniListAnimeImageProvider.cs

```diff
@@ -73,9 +73,8 @@ public async Task<IEnumerable<RemoteImageInfo>> GetImages(string aid, Cancellati
 
         public async Task<HttpResponseMessage> GetImageResponse(string url, CancellationToken cancellationToken)
         {
-            var httpClient = Plugin.Instance.GetHttpClient();
-
-            return await httpClient.GetAsync(url).ConfigureAwait(false);
+            using var httpClient = Plugin.Instance.GetHttpClient();
```

Comment:
```suggestion
            var httpClient = Plugin.Instance.GetHttpClient();
```

Reply:
Other reviewer requested to add `using`. I believe it doesn't cause issues if `using` is omitted since this uses `HttpClientFactory`, but I don't think it causes any harm either?

Reply:
From the documentation:

> HttpClient instances can generally be treated as .NET objects not requiring disposal. Disposal cancels outgoing requests and guarantees the given HttpClient instance can't be used after calling [Dispose](https://learn.microsoft.com/en-us/dotnet/api/system.idisposable.dispose). IHttpClientFactory tracks and disposes resources used by HttpClient instances.

Please follow our standard usage and do not dispose the http client.